classCreation.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Implement a Polygon class with the following properties:
+ * 1. A constructor that takes an array of integer side lengths.
+ * 2. A 'perimeter' method that returns the sum of the Polygon's side lengths.
+ */
+class Polygon{
+    
+    constructor(nums)
+    {
+        this.nums = nums;
+    }
+
+    perimeter()
```

Comment:
refactor this method. write a method signature where we can pass array as argument.